### PR TITLE
Mobile Menu

### PR DESCRIPTION
**Pull request by mgmediaweb**
# Generate Mobile Menu for portfolio

- Create a div 'mobile-menu' and position it in entire of the screen.
- Use UL/LI for the menu option.
- Use Close icon from FontAwesome libraries.
- Create an external javascript file linked to the original HTML.
- Create a JS function to show/hide the menu.
- Update the Linters libraries for check JS/CSS/HTML.
- All Linters checks passed.

### CODE REVIEW

**Review by nicupop729 — CHANGES_REQUESTED** [review]
Hi @mgmediaweb,
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request



You have submitted a project, that:

- [ ] Has linters disabled for the entire JavaScript file;
- [ ] Was not built in a pair-programming team as was requested by the project requirements;

Kindly, 

- [ ] Remove comments that disable linters for the entire JS file;
- [ ] As it is a milestone where we have to collaborate, please invite your partner to help you build your project and use the driver-navigator approach, where both of you need to be at a moment the driver and commit to this milestone;

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -28,7 +29,7 @@
                 </ul>
             </nav>
             <div class="movil-menu">
```
Comment (thecodechaser) [inline index.html:31]: - [ ] You did a good job on the functionalities but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design as much as possible. Please reduce the weight of the close(X) button and increase the font size for mobile menu options to make sure it matches the template.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @mgmediaweb, and @diegoh40👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Assets/css/style.css [hunk 2]
```diff
@@ -286,6 +365,34 @@ textarea {
   justify-content: space-between;
 }
 
+#mobile_menu {
```
Comment (Ismailco) [inline Assets/css/style.css:368]: - [ ] [optional] Your menu has some differences in the navigation links ```margin``` and the ```hr``` thickness, you can try to modify them so your design can be as close as possible to the Figma one

Screenshot_20220510_190024

**Review by Ismailco — APPROVED** [review]
### Hi @mgmediaweb,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally
✔️ Implement requirements

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**